drivers/mrf24j40/mrf24j40_netdev.c

```diff
@@ -106,6 +106,8 @@ static int _send(netdev_t *netdev, const struct iovec *vector, unsigned count)
         len = mrf24j40_tx_load(dev, ptr->iov_base, ptr->iov_len, len);
         if (i == 0) {
             dev->header_len = len;
```

Comment:
I did not look at this driver before, but that looks like an arbitrary assumption, that the header length is always the same as the length of the first iovec fragment.
Does not need to be addressed in this PR, but it's ugly..

Reply:
I agree, I think the correct solution here is to verify with the `len` whether these fields are in the current fragment.
I'll put this cleanup on my todo list for this driver.